CONTRIBUTING.md

```diff
@@ -0,0 +1,27 @@
+# Contributing
+
+Thank you for considering making contributions to Fourth State's Plasma MVP implementation! We welcome contributions from anyone! See the [open issues](https://github.com/FourthState/plasma-mvp-rootchain/issues) for things we need help with!
+
+## How to get started:
+
+Fork, then clone the repo:
```

Comment:
Once you fork, you should create a branch that is named after the feature you are trying to implement.
See the guideline posted here: https://nvie.com/posts/a-successful-git-branching-model/

Make a pull request from this branch into our master branch.